acq4/pyqtgraph/parametertree/parameterTypes.py

```diff
@@ -273,14 +273,17 @@ def optsChanged(self, param, opts):
         ParameterItem.optsChanged(self, param, opts)
         
         if 'readonly' in opts:
+            readonly = opts.pop('readonly')
```

Comment:
This is actually fixed upstream in pyqtgraph: https://github.com/pyqtgraph/pyqtgraph/blob/develop/pyqtgraph/parametertree/parameterTypes.py#L270

Let me see if I can get those changes merged in first..